app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -55,21 +55,12 @@ def new
     end
 
     def create
-      # Caching the original input params in case the form is not valid
-      original_input_params_for_form = params[hash_key_for_curation_concern].deep_dup
-      if create_work
-        after_create_response
+      case curation_concern
+      when ActiveFedora::Base
+        original_input_params_for_form = params[hash_key_for_curation_concern].deep_dup
+        actor.create(actor_environment) ? after_create_response : after_create_error(curation_concern.errors, original_input_params_for_form)
```

Comment:
No change to the processing for AF works.  Moved the code from `#create_work` to here so the `#create_valkyrie_work` method can focus on just valkyrie processing.  This also matches the pattern used for Collections.